holmes/plugins/toolsets/mcp/toolset_mcp.py

```diff
@@ -203,6 +247,11 @@ class RemoteMCPToolset(Toolset):
     icon_url: str = "https://registry.npmmirror.com/@lobehub/icons-static-png/1.46.0/files/light/mcp.png"
     _mcp_config: Optional[Union[MCPConfig, StdioMCPConfig]] = None
 
+    def get_context_fields(self) -> Optional[List[str]]:
+        if isinstance(self._mcp_config, MCPConfig):
```

Comment:
We intentionally didn't support stdio mcp?

Reply:
Thanks for pointing this out, I didn't realize that stdio has a different config class. Fixed it.